cheddar/cheddar-integration-aws/src/main/java/com/clicktravel/infrastructure/persistence/aws/dynamodb/DynamoDbTemplate.java

```diff
@@ -412,21 +415,38 @@ public void delete(final Item item, final PersistenceExceptionHandler<?>... pers
 
     @Override
     public <T extends Item> Collection<T> fetch(final Query query, final Class<T> itemClass) {
+        return fetch(query, itemClass, null);
+    }
+
+    @Override
+    public <T extends Item> Collection<T> fetch(final Query query, final Class<T> itemClass,
+            final Integer maxPageSize) {
```

Comment:
Always prefer `int` over `Integer` unless `null` has a meaning in the given context.